python/cudf/cudf/tests/conftest.py

```diff
@@ -176,6 +178,37 @@ def pytest_runtest_makereport(item, call):
     setattr(item, "report", {rep.when: rep})
 
 
+def _get_all_zones():
+    zones = []
+    for zone in zoneinfo.available_timezones():
+        # TODO: pandas 3.0 defaults to zoneinfo,
+        # so all_zone_names can use zoneinfo.available_timezones()
+        try:
+            pd.DatetimeTZDtype("ns", zone)
+        except Exception:
+            continue
+        else:
+            zones.append(zone)
+    return sorted(zones)
+
+
+# NOTE: _get_all_zones is a very large list; we likely do NOT want to
+# use it for more than a handful of tests
+@pytest.fixture(params=_get_all_zones())
+def all_timezones(request):
+    return request.param
+
+
+@pytest.fixture(
+    params=["America/New_York", "Asia/Tokyo", "CET", "Etc/GMT+1", "UTC"]
+)
+def limited_timezones(request):
+    """
+    Limited set of interesting timezones for testing.
```

Comment:
nitpick: By interesting I think you mean something like a small representative sample of all timezones
```suggestion
    Limited set of interesting timezones for testing.
```